.travis.yml

```diff
@@ -1,47 +1,41 @@
 language: python
+# This only controls the version of Python used to run tox, the
+# versions used for the tests are handled by tox itself. Python 3.5
+# is only installed if we explicitly request it, so we have to use
+# it as the base, so that it's available to tox. Python 2.7 and 3.4
+# are available in the Travis images by default. See:
+# https://github.com/travis-ci/travis-ci/issues/4794#issuecomment-143758799
+python: "3.5"
 
 sudo: false
 
 env:
     - TOX_ENV=py27-flake8
-    - TOX_ENV=py26-django1.5-drf2.3
-    - TOX_ENV=py26-django1.6-drf3.1
-    - TOX_ENV=py27-django1.5-drf2.3
-    - TOX_ENV=py27-django1.5-drf2.4
-    - TOX_ENV=py27-django1.5-drf3.0
-    - TOX_ENV=py27-django1.5-drf3.1
-    - TOX_ENV=py27-django1.6-drf2.3
-    - TOX_ENV=py27-django1.6-drf2.4
-    - TOX_ENV=py27-django1.6-drf3.0
-    - TOX_ENV=py27-django1.6-drf3.1
-    - TOX_ENV=py27-django1.7-drf2.3
-    - TOX_ENV=py27-django1.7-drf2.4
-    - TOX_ENV=py27-django1.7-drf3.0
-    - TOX_ENV=py27-django1.7-drf3.1
     - TOX_ENV=py27-django1.8-drf2.3
     - TOX_ENV=py27-django1.8-drf2.4
     - TOX_ENV=py27-django1.8-drf3.0
     - TOX_ENV=py27-django1.8-drf3.1
-    - TOX_ENV=py33-django1.5-drf2.3
+    - TOX_ENV=py27-django1.8-drf3.2
+    - TOX_ENV=py27-django1.8-drf3.3
+    - TOX_ENV=py27-django1.9-drf3.3
     - TOX_ENV=py33-django1.8-drf3.1
-    - TOX_ENV=py34-django1.5-drf2.3
-    - TOX_ENV=py34-django1.5-drf2.4
-    - TOX_ENV=py34-django1.5-drf3.0
-    - TOX_ENV=py34-django1.5-drf3.1
-    - TOX_ENV=py34-django1.6-drf2.3
-    - TOX_ENV=py34-django1.6-drf2.4
-    - TOX_ENV=py34-django1.6-drf3.0
-    - TOX_ENV=py34-django1.6-drf3.1
-    - TOX_ENV=py34-django1.7-drf2.3
-    - TOX_ENV=py34-django1.7-drf2.4
-    - TOX_ENV=py34-django1.7-drf3.0
-    - TOX_ENV=py34-django1.7-drf3.1
+    - TOX_ENV=py33-django1.8-drf3.2
+    - TOX_ENV=py33-django1.8-drf3.3
     - TOX_ENV=py34-django1.8-drf2.3
     - TOX_ENV=py34-django1.8-drf2.4
     - TOX_ENV=py34-django1.8-drf3.0
     - TOX_ENV=py34-django1.8-drf3.1
+    - TOX_ENV=py34-django1.8-drf3.2
+    - TOX_ENV=py34-django1.8-drf3.3
+    - TOX_ENV=py34-django1.9-drf3.3
+    - TOX_ENV=py35-django1.8-drf3.3
+    - TOX_ENV=py35-django1.9-drf3.3
 
 matrix:
+  allow_failures:
+    - env: TOX_ENV=py27-django1.9-drf3.3
+    - env: TOX_ENV=py34-django1.9-drf3.3
+    - env: TOX_ENV=py35-django1.9-drf3.3
```

Comment:
Agreed, but in a PR after this one to save blocking & to keep this PR smaller and easier to review. I'll file a separate issue for the Django 1.9 failures shortly - having the tests running in the meantime will allow others to iterate more easily for the 1.9 compat fixes.